internal/cli/install.go

```diff
@@ -367,5 +367,8 @@ deployments. If this is incorrect, manually set it using the CLI command
 
 Advertise Address: %[2]s
 Web UI Address: %[3]s
+
+To launch and authenticate into the Web UI, run:
+waypoint ui -authenticate
```

Comment:
Maybe this could move above here onto line 366? I kind of like the two addresses being the last thing you see in the output.

Reply:
changed!